packages/react-devtools-shared/src/backend/legacy/renderer.js

```diff
@@ -694,8 +694,9 @@ export function attach(
     requestID: number,
     id: number,
     path: Array<string | number> | null,
+    forceFullData: boolean,
```

Comment:
apologies if i've missed this context from before; what is the story behind the legacy renderer? how much do we need to support it? and is this also being exercised in the tests?

Reply:
Bare minimal support.

I didn't add a test for this because I thought it would be a big hassle 😅